src/Layer/Layer.js

```diff
@@ -52,17 +52,17 @@ class Layer extends THREE.EventDispatcher {
      * @param {Source|boolean} config.source - instantiated Source specifies data source to display.
      * if config.source is a boolean, it can only be false. if config.source is false,
      * the layer doesn't need Source (like debug Layer or procedural layer).
-     * @param {StyleOptions|Style} [config.style] - an object that contain any properties
+     * @param {StyleOptions} [config.style] - an object that contain any properties
      * (order, zoom, fill, stroke, point, text or/and icon)
      * and sub properties of a Style (@see {@link StyleOptions}). Or directly a {@link Style} .<br/>
      * When entering a StyleOptions the missing style properties will be look for in the data (if any)
      * what won't be done when you use a Style.
      * @param {number} [config.cacheLifeTime=Infinity] - set life time value in cache.
-     * This value is used for [Cache]{@link Cache} expiration mechanism.
-     * @param {(boolean|Object)} [config.addLabelLayer=false] - Used to tell if this layer has
+     * This value is used for cache expiration mechanism.
+     * @param {Object} [config.addLabelLayer] - Used to tell if this layer has
```

Comment:
Doesn't this needs to still be typed as `boolean|Object` since it still can be a boolean ?

Reply:
Does the `true` value is actually used in the code?
It seems the only usage of `addLabelLayer` if it is not `undefined` is in the code below and it is used as an object (a `true` value creates a type error) https://github.com/iTowns/itowns/blob/ff9a86adf64ef3626b5d3542ef2145b3fd8eda32/src/Core/View.js#L66-L84

Reply:
It is used in [this example](https://github.com/iTowns/itowns/blob/ff9a86adf64ef3626b5d3542ef2145b3fd8eda32/examples/source_file_geojson_raster_2.html#L107). It should not create a type error in the place you mentioned since `true.someProperty` will be evaluated as `undefined` ([here](https://github.com/iTowns/itowns/blob/ff9a86adf64ef3626b5d3542ef2145b3fd8eda32/src/Core/View.js#L79) and [here](https://github.com/iTowns/itowns/blob/ff9a86adf64ef3626b5d3542ef2145b3fd8eda32/src/Core/View.js#L83)). This is not a clean way of coding though, I admit it.

Reply:
Forgot that literal values could have method... Fixed it!